### ISSUE TITLE
Syntax Error

### ISSUE THREAD

**Issue by ghost** [peoplma/subredditarchive#3]
Most probably because of the ASCII art.
I know it makes the script look cool, but is it really needed?

**Comment by peoplma** [comment]
Nah, it's not needed at all, feel free to take it out.  It shouldn't give a syntax error though I don't think.  Which line did it get stuck on?

**Comment by ghost** [comment]
I don't know, but you should probably remove the ascii art.
I'm afraid that if I remove it myself a line gets deleted by mistake and then I'd have absolutely _no idea_ which line that was!

**Comment by ev88** [comment]
If you're using Python 3.x and you get a syntax error, it's likely related to the print statements not having parentheses.  These weren't required in Python 2.

http://stackoverflow.com/questions/25445439/what-does-syntaxerror-missing-parentheses-in-call-to-print-mean-in-python

I managed to fix the syntax errors but nothing seems to happen after I input a number at the first menu.
